index.html

```diff
@@ -0,0 +1,256 @@
+    <!---POPPINS-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+    <!--CRETE ROUND-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <!--INTER-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="wrap-section">
+        <header>
+            <nav>
+                <a href="#" class="my-logo">Aneal Laryea</a>
+                <i class="fa-solid fa-bars"></i>
+            </nav> 
+        </header>
+    
+        <section class="headline">
+            <div class="primary-text">
+                <h1>Hey There.
+                    I’m Aneal</h1>
+            </div>
+            <div class="supporting-text">
+                <h5>I’am a Software Developer</h5>
+            </div>
+            <div class="description">
+                <h5>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</h5>
+            </div>
+            <div class="social-media">
+                <ul class="socials">
+                    <li class="ss"><i class="fa-brands fa-github"></i></li>
+                    <li class="ss"><i class="fa-brands fa-linkedin"></i></li>
+                    <li class="ss"><i class="fa-brands fa-angellist"></i></li>
+                    <li class="ss"><i class="fa-brands fa-twitter"></i></li>
+                    <li class="ss"><i class="fa-solid fa-m"></i></li>
+                </ul>
+            </div>
+        </section>
+
+         <!--work section-->
+        <section class="work-section">
+            <!--frame 44-->
+            <div class="title">
+               <h1 class="title-text">My Recent Works</h1>
+           </div>
+           <div class="frame">
+               <div class="featured-img">
+                   <div class="img-holder"></div>
+                   <div class="right-block">
+                       <div class="title-post"><h1>Multi-Post Stories</h1></div>
+                       <div class="support-text"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p></div>
```

Comment:
- [ ] Kindly use the correct font family for your project as instructed in the project brief and as it is in the Figma template. Try as best as possible to follow the design given. 
- [ ] Kindly make all headings in the entire page have the Google font [Crete Round](https://fonts.google.com/specimen/Crete+Round?query=Crete+Round) - which you must import from Google font and make use of it.
- [ ] Kindly make all body texts of your page have the font [Poppins font from Google](https://fonts.google.com/specimen/Poppins?query=poppins)